src/query/ts/series.go

```diff
@@ -21,15 +21,18 @@
 package ts
 
 import (
+	"time"
+
 	"github.com/m3db/m3/src/query/models"
 )
 
 // Series is the public interface to a block of timeseries values.  Each block has a start time,
 // a logical number of steps, and a step size indicating the number of milliseconds represented by each point.
 type Series struct {
-	name string
-	vals Values
-	Tags models.Tags
+	resolution int
```

Comment:
Can we make this `time.Duration` instead of just an int?

It's a loss of precision to turn this into millis and seems more type safe to just use `time.Duration` everywhere rather than an `int` that is named `blahMillis`.

Reply:
Sure sounds good, was just doing it quick and dirty for the conversion to millis later on